examples/bzlmod/MODULE.bazel

```diff
@@ -32,4 +35,7 @@ buildbuddy = use_extension("@buildbuddy//:extensions.bzl", "buildbuddy")
 #     )
 
 # All relevant toolchain and platform targets should be available under "@buildbuddy_toolchain//..." repository
+#
+# For convenience, the platform and toolchain targets are also "alias" in the module under
+# "@toolchains_buildbuddy//platforms:all" and "@toolchains_buildbuddy//toolchains/cc:all" packages.
```

Comment:
Should we recommend those as the primary location? Then we wouldn't have to tell people about the extension repo and would be free to change its structure in the future. Users that migrate from WORKSPACE can already rename the repo with `repo_name`.

Reply:
Users could still bring in the extension repo under an arbitrary name with `use_repo`.

What do we gain by advertising both the module repo and the extension repo though? It seems more difficult to maintain both, so how about either just advertising the `alias` targets (don't tell users about the `use_repo`) or doing away with the `alias` targets (which requires committing to the extension repo name)?